Pipfile

```diff
@@ -28,6 +28,7 @@ aiida-vasp = "~=0.2.4"
 aiida-wannier90 = "~=2.0"
 aiida-yambo = "~=1.0"
 aiida-zeopp = "~=1.1"
+ase = "<3.20"
```

Comment:
Sorry, I just realized that we should probably add a comment here as to the reasoning for this constraint.

Reply:
added.